cli/src/commands/evolog.rs

```diff
@@ -151,14 +161,35 @@ pub(crate) fn cmd_evolog(
     if !args.no_graph {
         let mut raw_output = formatter.raw()?;
         let mut graph = get_graphlog(graph_style, raw_output.as_mut());
-        for commit in commits {
-            let edges = commit
-                .predecessor_ids()
+
+        let commit_dag: Vec<Result<GraphNode<Commit>, Infallible>> = commits
+            .iter()
```

Comment:
nit: `.into_iter()` instead of `.iter()` + `.clone()`

Reply:
I changed this instance and the other one you pointed out, thanks!